hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -951,13 +951,6 @@
     public static final HazelcastProperty DISCOVERY_SPI_PUBLIC_IP_ENABLED
             = new HazelcastProperty("hazelcast.discovery.public.ip.enabled", false);
 
-    /**
-     * When this property is true, if the server can not determine the connected client version, it shall assume that it is of
-     * 3.6.x version client. This property is especially needed if you are using ICache (or JCache).
-     */
-    public static final HazelcastProperty COMPATIBILITY_3_6_CLIENT_ENABLED
```

Comment:
There was a counterpart of this on the client side. That should be deleted also.

Reply:
OK. Just removed and pushed.